apps/aens/src/aens_hash.erl

```diff
@@ -28,14 +34,37 @@
 %%% API
 %%%===================================================================
 
+
 -spec commitment_hash(binary(), integer()) -> commitment_hash().
 commitment_hash(NameAscii, Salt) ->
+    case aens_utils:name_domain(NameAscii) of
+        {ok, Domain} when Domain =:= <<"aet">> ->
+            SaltBin = int_to_bin(Salt),
+            hash(<<NameAscii/binary, SaltBin/binary>>);
+        _ ->
+            %% This could be .test or any other wrong name backward compatible
+            pre_lima_commitment_hash(NameAscii, Salt)
+    end.
+
+-spec name_hash(binary()) -> name_hash().
+name_hash(NameAscii) ->
+    case aens_utils:name_domain(NameAscii) of
+        {ok, Domain} when Domain =:= <<"aet">> ->
+            hash(NameAscii);
+        _ ->
+            %% This could be .test or any other wrong name backward compatible
+            pre_lima_name_hash(NameAscii)
+    end.
+
+
+-spec pre_lima_commitment_hash(binary(), integer()) -> commitment_hash().
+pre_lima_commitment_hash(NameAscii, Salt) ->
     NameHash = name_hash(NameAscii),
```

Comment:
Shouldn't this call `pre_lima_name_hash` instead?

Reply:
it will automatically take that one, but sure, that could save a function call. Well spotted, shall fix it for next rebase